core/modules/system/tests/cron_queue_test/cron_queue_test.info

```diff
@@ -1,7 +1,7 @@
-name = "Cron Queue test"
-description = "Support module for the cron queue runner."
-backdrop = 1.x
-package = Testing
-version = BACKDROP_VERSION
 type = module
+name = Cron Queue test
```

Comment:
Trivial, I know, but I preferred it when type was listed last.

Reply:
Fine by me. That's just how it was in the (not quite duplicate) file that was being loaded. Now fixed.